case_utils/case_validate/__init__.py

```diff
@@ -36,11 +36,12 @@
 import logging
 import os
 import sys
-import typing
 import warnings
+from typing import Any, Dict, List, Optional, Set, Tuple, Union
```

Comment:
I'm happy with this vote to import `typing` symbols.  This can come in its own little code-style PR.

Reply:
The updated PR has been included (with newly used symbols)